typings/browser/ambient/es6-shim/es6-shim.d.ts

```diff
@@ -1,670 +0,0 @@
-// Compiled using typings@0.6.8
```

Comment:
I think the typings package for es6-shim (and the format of typings in general) has been updated to no longer use "ambient" and instead use "global" terminology. That's why this file has now been moved to the global folder and renamed to index.d.ts.

I couldn't find a good reference for when this changed, but I did confirm that if I create a fresh app via "ionic start myApp --v2 --ts" then this file is removed and the typings/global/es6-shim/ folder is created (with the index.d.ts and typings.json)